ci/build_docs.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key docs \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
+
+rapids-mamba-retry env create --force -f env.yaml -n test
+conda activate test
```

Comment:
The `test` environment name makes more sense for test jobs. Should we rename it to `docs`?
```suggestion
rapids-mamba-retry env create --force -f env.yaml -n docs
conda activate docs
```